etc/inc/system.inc

```diff
@@ -2270,4 +2270,18 @@ function system_get_dmesg_boot() {
 	return file_get_contents("{$g['varlog_path']}/dmesg.boot");
 }
 
+function get_posible_unbound_local_zone_types () {
```

Comment:
s/posible/possible
And same change where the function is used.
Otherwise the mis-spelling will trip up English speakers for years to come, like HTTP "referer" became entrenched in a standard :)